openadapt/scripts/scrub.py

```diff
@@ -0,0 +1,104 @@
+"""Module for scrubbing a media file.
+
+Usage: python -m openadapt.scripts.scrub <media_file_path>
+
+"""
+
+import os
+import sys
+
+from loguru import logger
+from PIL import Image
+import cv2
+import numpy as np
+
+from openadapt import scrub
+
+
+def scrub_mp4(mp4_file: str) -> str:
+    """
+    Scrub a mp4 file.
+
+    Args:
+        mp4_file: Path to the mp4 file.
+
+    Returns:
+        Path to the scrubbed (redacted) mp4 file.
+    """
+
+    scrubbed_file = mp4_file[:-4] + "_scrubbed.mp4"
+
+    cap = cv2.VideoCapture(mp4_file)
+    frame_width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    frame_height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+    fps = int(cap.get(cv2.CAP_PROP_FPS))
+    fourcc = cv2.VideoWriter_fourcc("m", "p", "4", "v")
+    out = cv2.VideoWriter(
+        scrubbed_file, fourcc, fps, (frame_width, frame_height)
+    )
+
+    while cap.isOpened():
```

Comment:
Any way to print progress? It would be nice to know how much has been complete and how much is left 🙏

Reply:
Trying `tqdm` to accomplish the above requirement! :)

A commit will follow

Reply:
A demo on slack: https://mldsai.slack.com/files/U056HTNBQ5S/F05DFG8MQ76/scrub.py_-_openadapt_-_visual_studio_code_-_insiders_2023-06-20_12-09-42.mp4

Reply:
Addressed in: [93b04d1](https://github.com/MLDSAI/OpenAdapt/pull/292/commits/93b04d1ac6193c297703c94bcfdb70f03faec9bb)

Reply:
![Progress Bar Demo](https://github.com/MLDSAI/OpenAdapt/assets/65433817/3a32b2da-afef-4afe-9c4d-be98fe3eb87a)